nodetool

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+# If STARGATE_HOME wasn't specified in the environment, then search for one...
+if [ "x$STARGATE_HOME" == "x" ]; then
+    STARGATE_HOME=`dirname "$0"`
+    if [[ ! -d $STARGATE_HOME ]]; then
+      echo "STARGATE_HOME not found"
+      exit 2
+    fi
+else
+    if [[ ! -d $STARGATE_HOME ]]; then
+      echo "STARGATE_HOME: '$STARGATE_HOME' is not a directory"
+      exit 2
+    fi
+fi
+
+declare -a CMD=("${STARGATE_HOME}/starctl" "--nodetool")
+
+# If the Stargate LISTEN address is set use it for the --host nodetool option by default
+if [[ ! -z "$LISTEN" ]]; then
+    CMD+=("-T" "--host" "-T" "$LISTEN")
+fi
```

Comment:
Would this make sense to do in Starter.java instead since its typically not set in docker?

```
    if [[ ! -z "$LISTEN" ]]; then
        STARGATE_ARGS+=("--listen" "$LISTEN")
    else
        LISTEN="$(_ip_address)"
        STARGATE_ARGS+=("--listen" "$LISTEN")
    fi
```

Reply:
Probably... I'll give it a try.